back/spec/acceptance/resident_native_survey_responses_spec.rb

```diff
@@ -385,9 +387,8 @@
             expect(IdeaFile.count).to eq 1
 
             # Verify that the custom field value is still the existing referenced file.
-            expect(input.reload.custom_field_values).to eq({
-              'custom_field_name2' => { 'id' => existing_file.id, 'name' => 'existing_file.pdf' }
-            })
+            expect(input.reload.custom_field_values['custom_field_name2'])
+              .to eq({ 'id' => existing_file.id, 'name' => 'existing_file.pdf' })
```

Comment:
@jamesspeake Following [this commit](url), I found that this test was failing with:
```
Failure/Error:
       expect(input.reload.custom_field_values).to eq({
         'custom_field_name2' => { 'id' => existing_file.id, 'name' => 'existing_file.pdf' }
       })
     
       expected: {"custom_field_name2"=>{"id"=>"5ec099a2-135c-4c3c-8174-1c3e5c771a95", "name"=>"existing_file.pdf"}}
            got: {"custom_field_name1"=>"Cat", "custom_field_name2"=>{"id"=>"5ec099a2-135c-4c3c-8174-1c3e5c771a95", "name"=>"existing_file.pdf"}}
 ```
 In other words, before I changed this test, it seems it was overwriting the `Idea.custom_field _values` with the new key/value provided on update, whereas now it is inserting the new key/value alongside the existing key/value.
 
 Is this OK, or have I broken something here?
 Either result seems to satisfy the intention expressed by the comment
 ```
 # Verify that the custom field value is still the existing referenced file.
 ```
 and I can' think of a reason why this behaviour would be problematic. WDYT?